airflow-core/src/airflow/api_fastapi/core_api/routes/public/hitl.py

```diff
@@ -260,7 +281,7 @@ def get_hitl_details(
     readable_ti_filter: ReadableTIFilterDep,
     session: SessionDep,
 ) -> HITLDetailCollection:
-    """Get Human-in-the-loop details."""
+    """Get all Human-in-the-loop details."""
```

Comment:
We're going to support the filter. so the original docstring might be more accurate

Reply:
Changed it.